hw/ip/dma/data/dma.hjson

```diff
@@ -315,7 +315,11 @@
     { name: "CFG_REGWEN"
       desc: '''
             Indicates whether the configuration registers are locked because the DMA controller is operating.
-            The CONTROL, STATUS, and CLEAR_STATE registers remain usable.
+            In the idle state, the this register is set to kMultiBitBool4True.
+            When the DMA is performing an operation, i.e., the DMA is busy, this register is set to kMultiBitBool4False.
```

Comment:
Is it equivalent to `STATUS.busy` then? In this case, I'm not sure to understand the added value of having a read-only multibit bool here. If for some reason this bit was corrupted, the HW would reject a SW change anyway isn't it?

Reply:
> Is it equivalent to STATUS.busy then? 

Yes, to the inverse.

The Read-only REGWEN is only needed to implement the lock for the register set, such that it can be used for the register generation tool. I agree that a single flip on busy would lock up the registers. However, in that case, the abort feature can be used to reset busy back to zero so that the DMA can be programmed again. Previously, there was a dedicated REGWEN register, that got set by a 1-bit condition, which is now different. A 1-bit signal is combinationally used to compute the multi-bit value and the lock. Probably the multi-bit value does not add any security here.

Reply:
I just read in the OTP controller that a very similar feature is implemented through the `DIRECT_ACCESS_REGWEN` register (`hw/ip/otp_ctrl/data/otp_ctrl.html#direct_access_regwen`) but exposed as a single bit value. In a future implementation of the DMA, I think it could be nice if both IPs use a single bit.